cli/src/cli.rs

```diff
@@ -114,10 +114,6 @@ pub struct RunCmd {
 	#[arg(long = "grandpa-pause", num_args = 2)]
 	pub grandpa_pause: Vec<u32>,
 
-	/// Enable the BEEFY gadget (only on Rococo or Wococo for now).
-	#[arg(long)]
-	pub beefy: bool,
```

Comment:
I'd still keep the flag but defaulting to `true` only on testnets just in case we hit a snag and want a quick fix. At least for a bit more time.

Reply:
Sounds like a good idea.

Reply:
Done